### PR TITLE
SAK-31833 fix error in logs (rwiki)

### PR DESCRIPTION
**Pull request by bjones86**
https://jira.sakaiproject.org/browse/SAK-31833

cc: @juanjmerono

### CODE REVIEW

File: rwiki/rwiki-util/radeox/src/java/org/radeox/filter/XHTMLFilter.java [hunk 1]
```diff
@@ -149,7 +149,10 @@ public String filter(String input, FilterContext context)
 			String output = new String(baos.toByteArray(), "UTF-8");
 			int startBlock = output.indexOf("<sr>");
 			int endBlock = output.indexOf("</sr>");
-			finalOutput = output.substring(startBlock + 4, endBlock);
+			if(startBlock >= 0 && endBlock >= 0)
+			{
+				finalOutput = output.substring(startBlock + 4, endBlock);
+			}
 			// log.warn("Output is "+finalOutput);
```
Comment (bjones86) [inline rwiki/rwiki-util/radeox/src/java/org/radeox/filter/XHTMLFilter.java]: Yep, will do. Thanks @ern

**Comment by bjones86** [comment]
Updated based on @ern's comment, also added a bit more range checking in the conditional.

**Comment by buckett** [comment]
I'm slightly confused about the extra checking as `indexOf` will only return a range inside the string or `-1`

**Comment by bjones86** [comment]
Yes indeed, thanks @buckett; apparently I wasn't thinking clearly yesterday.

**Comment by buckett** [comment]
@bjones86 All magic.

**Pull request merged**